src/plugins/NEMD/MettDeamon.cpp

```diff
@@ -1678,15 +1629,15 @@ void Reservoir::readFromFileBinaryHeader()
 	XMLfileUnits inp(_filepath.header);
 
 	if(not inp.changecurrentnode("/mardyn")) {
-		global_log->error() << "Could not find root node /mardyn in XML input file." << endl;
-		global_log->fatal() << "Not a valid MarDyn XML input file." << endl;
+		global_log->error() << "[MettDeamon] Could not find root node /mardyn in XML header file or file itself." << endl;
+		global_log->fatal() << "[MettDeamon] Not a valid MarDyn XML header file." << endl;
 		Simulation::exit(1);
 	}
 
 	bool bInputOk = true;
 	double dCurrentTime = 0.;
 	double dBL[3];
-	uint64_t nNumMols;
+	uint64_t nNumMols = 0;
```

Comment:
Isn't the `unsigned long` characteristic not already given by `uint64_t` and therefore doubles the definition (-> error-prone)?

Reply:
technically you invoke a cast because you pass an int (`0`) to an `unsigned long`. Not really a problem here but just a good habit to use curly braces that force you to use the correct type.
https://stackoverflow.com/questions/18222926/why-is-list-initialization-using-curly-braces-better-than-the-alternatives